docs/intro_notebook.ipynb

```diff
@@ -135,27 +138,26 @@
   },
   {
    "cell_type": "markdown",
-   "id": "99453d48",
+   "id": "5ef6f27b",
    "metadata": {},
    "source": [
-    "## Track custom functions\n",
+    "## Vendor Integrations (OpenAI, Anthropic, Mistral, etc...)\n",
     "\n",
-    "Add the @weave.op decorator to the functions you want to track"
+    "Here, we're automatically tracking all calls to `openai`. We automatically track a lot of LLM libraries, but it's really easy to add support for whatever LLM you're using, as you'll see below. "
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "a5a1817e",
+   "id": "ac8a2b59",
    "metadata": {},
    "outputs": [],
    "source": [
     "import weave\n",
-    "from openai import OpenAI\n",
     "\n",
     "weave.init(PROJECT)\n",
     "\n",
-    "@weave.op\n",
+    "@weave.op()\n",
```

Comment:
Is there a reason to put the parens in? I think we've been moving towards stripping them out.

Reply:
right now this is actually broken with code capture :/